src/options/options.html

```diff
@@ -14,6 +14,10 @@
                 <label>Open as popup:</label>
                 <input id="gitpod-open-as-popup" type="checkbox"/>
             </div>
+            <div class="row">
+                <label>Replace <kbd>.</kbd> keybind (opens Gitpod instead of github.dev):</label>
```

Comment:
**thought**: Although this could be out of the scope of this PR in the spirit of MVC (minimum viable change), we could also extend this functionality also for GitLab which also adopted a similar approach where the `.` keybind opens the WebIDE[[1](https://docs.gitlab.com/ee/user/project/web_ide/)] in GitLab.

Reply:
Implemented in [`c7b2cdd` (#54)](https://github.com/gitpod-io/browser-extension/pull/54/commits/c7b2cddf9b45df8fe8704ee9403c2f83d30f4478).